### ISSUE TITLE
Change backlight brightness according to time of day/sun

### ISSUE THREAD

**Issue by allanlaal** [polychromatic/polychromatic#423]
### Details about the problem

I want the kb backlight to correspond on how bright it is outside to increase usability

since there isn't a brightness sensor in my kb, its best to set this using sun times (I live in scandinavia, so we don't have fixed sunset/sunrise times)

good library: https://github.com/risacher/sunwait

**Comment by lah7** [comment]
Thanks for the suggestion. That would be a good enhancement for the [planned "Triggers" feature](https://docs.polychromatic.app/roadmap/). To start with, I was thinking based on time (#272).

[`sunwait`](https://risacher.org/sunwait/) appears to be a C++ program, not a Python library. It seems like it can be used by creating a simple `brightness.sh` script and making this run as soon as you login (in "Startup Applications") like so:

```bash
#!/bin/bash
while True:
    sunwait wait rise
    polychromatic-cli -o brightness -p 80
    sunwait wait set
    polychromatic-cli -o brightness -p 30
```

☝🏼 That would run as a loop endlessly in the background.

**Comment by wael444** [comment]
Isn't the correct Bash syntax this?
```bash
while true; do
    sunwait wait rise
    polychromatic-cli -o brightness -p 80
    sunwait wait set
    polychromatic-cli -o brightness -p 30
done
```
?
You can also substitute `true` with `:`

**Comment by lah7** [comment]
Yes, good catch, thanks! Yours is correct. I got confused with Python's syntax. I've edited my comment to fix up.
